### ISSUE TITLE
[1.1.5] Inventory sync blocked indefinitely if initial one fails

### ISSUE THREAD

**Issue by jhoule86** [wayfair-contribs/plentymarkets-plugin#204]
**Date(s) issue was encountered**: December 2020
**Supplier's client ID (if known):** N/A
**Wayfair plugin Version (displayed in `Deployed` column in the details of a Plentymarkets Plugin Set):** 1.1.5 only
**Plentymarkets Version / release channel:** 7

**Short description:**
Inventory synchronization attempts are halted if the initial inventory sync (after installation/upgrade) fails to complete.
"NoReferencePointForPartialInventorySyncException" may be appearing in logs every 15 minutes.

**To Reproduce**
1. Install / upgrade the Wayfair plugin to **version 1.1.5**
2. Cause / wait for the initial "Full Inventory Synchronization" to fail
3. Wait at least 10 hours
4. Visiting the plugin's `Inventory` page **OR** wait for the automated Inventory Synchronization event to start 24 hours after the first attempt

**Expected behavior:**
The Wayfair plugin should detect that the **initial** Inventory Synchronization failed to be completed, and try to start the process again.

**Actual behavior:**
The Wayfair plugin throws an `InventorySyncInProgressException` and prevents any subsequent Inventory Synchronization activities.

**Screenshots:**
![image](https://user-images.githubusercontent.com/44010975/102269488-b4b59300-3eea-11eb-8005-6b525054ba42.png)

**Related logs:**
- "Inventory Errors"
- "NoReferencePointForPartialInventorySyncException"

**Comment by jhoule86** [comment]
It may be very difficult to solve the root cause of the failed initial inventory sync, but we should be able to unblock the _attempts_ to try again, which will provide more accurate situational reporting in the Wayfair logs on Plentymarkets systems.

**Comment by jhoule86** [comment]
**NOTICE:** fact that the Initial Inventory Sync is failing is a separate issue with a separate cause.
Wayfair and Plentymarkets will continue to investigate Inventory Sync stability issues.
